misc/gen-syntax.py

```diff
@@ -182,19 +183,25 @@ def pad_prefix(txt):
     return txt + (" " * (32 - len(txt)))
 
 
+def write_output(output, txt):
+    args.output.write(txt + "\n")
+    logger.debug(txt)
+
+
 if __name__ == "__main__":
     logging.basicConfig(level=logging.INFO)
 
-    import argparse
-
     p = argparse.ArgumentParser()
     p.add_argument("input", type=argparse.FileType("r"))
     p.add_argument("version")
-    a = p.parse_args()
+    p.add_argument("output", nargs="?", type=argparse.FileType("w"),
+                   default=os.path.join("..", "syntax", "sqf.vim"))
```

Comment:
This will be relative to the current working directory, not the file. So it could have surprising behaviour. I think stdout is a more better default because it has fewer side-effects (`default="-"`).

Reply:
Sure, works for me. Will do changes once #4 is addressed to not mix it up too much.